From cff696db91319ee641933117be291e2a9075b554 Mon Sep 17 00:00:00 2001
From: Register <458892+aieu@users.noreply.github.com>
Date: Mon, 17 Sep 2001 00:00:00 +0800
Subject: [PATCH] uboot: increase voltage for sgmii

Signed-off-by: Register <458892+aieu@users.noreply.github.com>
---
 arch/arm/dts/armada-3720-catdrive.dts | 1 +
 1 file changed, 1 insertion(+)

diff --git a/arch/arm/dts/armada-3720-catdrive.dts b/arch/arm/dts/armada-3720-catdrive.dts
index a4a4de7..b734fc8 100644
--- a/arch/arm/dts/armada-3720-catdrive.dts
+++ b/arch/arm/dts/armada-3720-catdrive.dts
@@ -30,6 +30,7 @@
 				reg = <0x32004 0x04>;
 				ethphy0: ethernet-phy@0 {
 					reg = <0>;
+					marvell,reg-init = <0x01 0x1A 0x0 0x47>;
 				};
 			};
 		};
-- 
2.26.3

